deploy/lib/server_config.rb

```diff
@@ -104,6 +104,9 @@ def initialize(options)
     else
       @qconsole_port = @bootstrap_port
     end
+
+    r = execute_query %Q{xdmp:host-name()}
+    @properties["ml.server-name"] = parse_body(r.body)
```

Comment:
Can you elaborate? Keep in mind server-name is only used within ml-config. We will remain using ml.server to make http/xcc calls..